rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/stop_scan/action.py

```diff
@@ -0,0 +1,25 @@
+import insightconnect_plugin_runtime
+
+from icon_rapid7_insightvm_cloud.util.error_handling import ERRORS
+from .schema import StopScanInput, StopScanOutput, Input, Output
+
+# Custom imports below
+
+
+class StopScan(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="stop_scan",
+            description="Stop a currently running scan",
+            input=StopScanInput(),
+            output=StopScanOutput(),
+        )
+
+    def run(self, params={}):
+        scan_id = params.get(Input.ID)
+        url = f"scan/{scan_id}/stop"
+        response = self.connection.ivm_cloud_api.call_api(url, "POST")
+        if response == 202:
```

Comment:
This is what made me think we should rework the call_api function a bit. In other places the full response body is returned.

Reply:
The problem is that for stop_scan there is no response body at all. It only returns a status code and nothing else.